### PR TITLE
Draft new Output section for VS Code

### PR DESCRIPTION
**Pull request by micgibso**
Signed-off-by: micgibso <micgibso@uk.ibm.com>

New Output section for https://github.com/eclipse/codewind/issues/1126

@tetchel Please review.

### CODE REVIEW

File: docs/_documentations/mdt-vsc-troubleshooting.md [hunk 1]
```diff
@@ -18,6 +18,18 @@ You can also post questions on our [Mattermost channel](https://mattermost.eclip
 
 ***
 
+### **Codewind Output stream**
+
+The Codewind Output stream is a tab within Codewind that is visible in the VS Code editor. It logs `cwctl` commands together with their ouput, for example:
+
+![Image of VS Code output](dist/images/cdt-vsc/output.png)<br>
```
Comment (tetchel) [inline docs/_documentations/mdt-vsc-troubleshooting.md]: I don't think we need both images, we can just show the second one.

File: docs/_documentations/mdt-vsc-troubleshooting.md [hunk 2]
```diff
@@ -18,6 +18,18 @@ You can also post questions on our [Mattermost channel](https://mattermost.eclip
 
 ***
 
+### **Codewind Output stream**
+
+The Codewind Output stream is a tab within Codewind that is visible in the VS Code editor. It logs `cwctl` commands together with their ouput, for example:
```
Comment (tetchel) [inline docs/_documentations/mdt-vsc-troubleshooting.md]: > is a tab within Codewind that is visible in the VS Code editor.

It is not a tab, and it is not "within Codewind". The vs code term _is_ output stream. So I think

`The Codewind Output stream is now available in the VS Code editor.`

or similar, is sufficient.

File: docs/_documentations/mdt-vsc-troubleshooting.md [hunk 3]
```diff
@@ -18,6 +18,18 @@ You can also post questions on our [Mattermost channel](https://mattermost.eclip
...
+
+![Image of VS Code output](dist/images/cdt-vsc/output.png)<br>
+
+Check the Output stream first when troubleshooting because it is particularly useful in helping you to debug unusual problems especially when starting Codewind. Some errors will also provide you with a button to open the Output stream, for example:
```
Comment (tetchel) [inline docs/_documentations/mdt-vsc-troubleshooting.md]: I'd prefer to call it "the Codewind output stream" here

File: docs/_documentations/mdt-vsc-troubleshooting.md (hunk 1 above)
Comment (micgibso) [reply docs/_documentations/mdt-vsc-troubleshooting.md]: Removed the first, and tweaked the words accordingly.

File: docs/_documentations/mdt-vsc-troubleshooting.md (hunk 2 above)
Comment (micgibso) [reply docs/_documentations/mdt-vsc-troubleshooting.md]: Reworded.

File: docs/_documentations/mdt-vsc-troubleshooting.md (hunk 3 above)
Comment (micgibso) [reply docs/_documentations/mdt-vsc-troubleshooting.md]: Corrected

**Comment by micgibso** [comment]
@tetchel Hi Tim, thanks, review comments incorporated.

**Pull request merged**